iFixit/essence
# Vimeo: Use https

Something on vimeo's side changed where they no longer accept http.  Let's update the api url to be https instead of HTTP.

QA:
--
Update `fg/essense` to be this branch. Then make sure vimeo uploads work.

cc @batbattur @sctice-ifixit 

Connects: https://github.com/iFixit/ifixit/issues/55404
